docs/javascript/v2/advanced-features/peer-connection-quality.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Peer Connection Quality
+nav: 5.06
+---
+
+You can get a score of current user's connection quality when in preview, post join connection quality will be available for all peers.
+The score ranges from 1-5, 5 being the hightest(good network health) and 1 being the lowest(bad network health).
+-1 indicates the sdk is not able to determine the network health within reasonable time.
+
```

Comment:
another section for "Measure Network Quality in Preview" can be added

Reply:
with a brief description about how it works from - https://www.100ms.live/docs/android/v2/features/Network-Quality-Reports